src/theme.js

```diff
@@ -24,12 +36,12 @@ const theme = createMuiTheme({
       dark: "#EF8341",
     },
     text: {
-      primary: "#6B5C43",
+      primary: "#6E661B",
     },
   },
   typography: {
     fontFamily: [
-      '"Libre Franklin"',
+      '"Arial Rounded MT Bold"',
```

Comment:
So this font is available on all systems?

Reply:
It's available on all Windows systems at the very least.

I went ahead and found the .woff and threw it in there anyways as a fallback in case the user's system doesn't have Arial Rounded MT Bold. It'll first see if the user has the font locally, and if not, it'll serve our .woff file.